Server/api/routes/login/osu.ts

```diff
@@ -127,7 +127,7 @@ osuRouter.get("/callback", async (ctx: ParameterizedContext<any>, next) => {
         const beatmaps = (await Axios.get(`https://osu.ppy.sh/api/get_beatmaps?k=${config.osu.v1.apiKey}&u=${ctx.state.user.osu.userID}`)).data;
         if (beatmaps.length != 0) {
             for (const beatmap of beatmaps) {
-                if (!beatmap.version.includes("'") && (beatmap.approved == 2 || beatmap.approved == 1)) {
```

Comment:
seemingly arbitrary checks like these could use a comment describing their intent. or, since it's repeated, be an appropriately-named helper function